addons/website_event_track/static/src/js/event_track_timer.js

```diff
@@ -0,0 +1,87 @@
+odoo.define('website_event_track.website_event_track_timer', function (require) {
+
+'use strict';
+
+const publicWidget = require('web.public.widget');
+
+/*
+ * Simple implementation of a timer widget that uses a "time to live" configuration
+ * value to countdown seconds on a target element.
+ * Will be used to visually countdown the time before a talk starts.
+ * When the timer reaches 0, the element destroys itself.
+ */
+publicWidget.registry.websiteEventTrackTimer = publicWidget.Widget.extend({
+
+    selector: '.o_we_track_timer',
+    events: {
+        'click .close': '_onCloseClick',
+    },
+
+    /**
+     * @override
+     */
+    start: function () {
+        return this._super.apply(this, arguments).then(() => {
+            let timeToLive = this.$el.data('time-to-leave');
```

Comment:
timeToLive vs time-to-leave?

Reply:
I mean sure, I'm not talking about the "-" notation VS camelCase, that part is correct, I'm talking about l*ea*ve vs l*i*ve ?
Or maybe I did not understand your answer?